README.md

```diff
@@ -34,128 +79,86 @@ Als Partner können sowohl Personen als auch Organisationen angelegt werden. Die
-  "email" : "...",  
-  "externePartnerId" : "...", 		// Eine beliebige, extern erzeugte Id. Z.B. SAP oder CRM Nummer.
-  "faxnummer" : "...",
-  "firmenname" : "...",
-  "firmennameZusatz" : "...",
-  "fusszeileFuerAussenauftritt" : "...", 		// mit \n  als Zeilentrenner
-  "gesperrt": false,            	      		// default: false 
-  "gesperrtTransitiv": false,	          		// true, wenn ein übergeordneter Partner gesperrt ist. (readonly)
-  "id":"...",                 					// EUROPACE 2 PartnerId (readonly)
-  "telefonnummer" : "...",
-  "typ": "ORGANISATION",            
-  "webseiteUrl" : "...",
-  "gehoertZuKreditinstitut" : false   // markiert Vertriebe, die einem Kreditinstitut zugeordnet sind
+    "partnerId": "...",
+    "name": "...",
+    "typ": "ORGANISATION",      
+    "email": "...",
+    "avatar": "...",
+    "gesperrt": false,
+    "parent": {
+        "partnerId": "..."
+    },
+    "telefonnummer": "...",
+    "firmenname": "...",
+    "webseite": "...",
+    "anschrift": {
+        "strasse": "...",
+        "hausnummer": "...",
+        "plz": "...",
+        "ort": "..."
+    },
+    "bankverbindung": {
+        "kontoinhaber": "...",
+        "bic": "...",
+        "iban": "..."
+    }
 }
 ```
 Dabei gilt:
 - Die Reihenfolge der Attribute spielt keine Rolle.
 - Die Attributbezeichner sind case-sensitive.
-- Zeilenumbrüche in "fusszeileFuerAussenauftritt" können durch '\n' erreicht werden.
-
-
-
-## Die HTTP Schnittstelle
-
-Die HTTP Schnittstelle der PEX-API ist unter der Basis-Url
-```
-https://www.europace2.de/partnermanagement/partner/
-```
-erreichbar.
-
-
-### Authentifizierung
-
-Für jeden Request ist eine Authentifizierung erforderlich. Die Authentifizierung erfolgt über HTTP Header.
-
-Request Header Name | Beschreibung
------------------|-------------
-X-ApiKey         | API Key des Benutzers / der Organisation
-X-PartnerId      | Europace2 PartnerId des Aufrufers
-
-Den API Key erhalten Sie von Ihrem Ansprechpartner bei EUROPACE 2. Die PartnerId finden Sie in den Einstellungen des jeweiligen Partners in der Weboberfläche des Partnermanagements.
-
-Schlägt die Authentifizierung fehl, erhält der Aufrufer eine HTTP Response mit Statuscode **401 UNAUTHORIZED**.
-
+- Zeilenumbrüche können durch '\n' erreicht werden.
 
 ### TraceId zur Nachverfolgbarkeit von Requests
 
-Für jeden Request sollte eine eindeutige id (TraceId) generiert werden, die den Request im EUROPACE 2 System nachverfolgbar macht und so bei etwaigen Problemen oder Fehlern die systemübergreifende Analyse erleichtert.
-Die Übermittlung der TraceId erfolgt über einen HTTP Header und wird als solcher auch in der Response zurückgeliefert. Wird keine TraceId übermittelt, enthält die Response eine in EUROPACE 2 automatisch erzeugte TraceId. 
+Für jeden Request sollte eine eindeutige id (TraceId) generiert werden, 
+die den Request im EUROPACE 2 System nachverfolgbar macht und so bei 
+etwaigen Problemen oder Fehlern die systemübergreifende Analyse erleichtert.
```

Comment:
```suggestion
Für jeden Request sollte eine eindeutige id (TraceId) generiert werden, die den Request im EUROPACE 2 System nachverfolgbar macht und so bei etwaigen Problemen oder Fehlern die systemübergreifende Analyse erleichtert.
```

Reply:
Verstehe den Änderungswunsch hier nicht ... ist die Einrückung anders? Den gesamten Abschnitt in eine Zeile pressen?

Reply:
Wenn du dir den rich text vergleich anschaust, dann ist das sowieso schon so, denn markdown erkennt deine Zeilenumbrüche nicht. Also spiegelt das hier nur mehr wieder wie es auch gerendert wird :)